lib/libmbedtls/mbedtls/include/mbedtls/bignum.h

```diff
@@ -199,6 +206,18 @@ extern void *mbedtls_mpi_mempool;
 void mbedtls_mpi_init( mbedtls_mpi *X );
 void mbedtls_mpi_init_mempool( mbedtls_mpi *X );
 
+/**
+ * \brief           Initialize one MPI with supplied value. The size of
+ *		    the MPI is limited to max @alloc_size.
+ *		    It's valid to call free on @X, but no memory is
+ *		    leaked if not. Calling free on @X will not affect the
+ *		    content of the @p array.
+ *
+ * \param X         One MPI to initialize.
+ */
+void mbedtls_mpi_init_static( mbedtls_mpi *X , mbedtls_mpi_uint *p,
+                              int sign, size_t alloc_size, size_t nblimbs);
```

Comment:
speca before `)`. Note several occurrences of missing space char near parenthesis.